src/components/Gallery/BaseImage.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { forwardRef, useEffect, useState } from 'react';
+import clsx from 'clsx';
+import { useTranslationContext } from '../../context';
+
+const DefaultImageFallback = () => {
+  const { t } = useTranslationContext('DefaultImageFallback');
+  return (
+    <div
+      className='str-chat__image-fallback'
+      data-testid='str-chat__image-fallback'
+      title={t<string>('Failed to load the image')}
```

Comment:
I think title should be/contain the image's name, otherwise it's to know which image can't be previewed. Also, I think "Failed to load the image" is misleading, users can think it failed to upload the image, even though only the image preview failed.

<img width="735" alt="Screenshot 2023-11-24 at 11 42 54" src="https://github.com/GetStream/stream-chat-react/assets/6690098/28751a1a-b02c-4574-832c-0e3f47dcae56">

Reply:
Yes, this is true, I will update

Reply:
@szuperaz I have used image names for titles and alts